### PR TITLE
Fix test for hocr-eval-lines

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by stweil** [comment]
Still buggy, see https://travis-ci.org/UB-Mannheim/hocr-tools/builds/162001129 :-(

**Comment by zuphilip** [comment]
Hm... I tested it in the docker container and the tests worked fine there.

**Comment by stweil** [comment]
The test now works with Python 2.7, but still fails with Python 3.4, see [Travis results](https://travis-ci.org/UB-Mannheim/hocr-tools/builds/162123974).

**Comment by kba** [comment]
It's fixed now for 3.4 as well.

**Comment by zuphilip** [comment]
Thank you very much!

File: hocr-eval-lines [hunk 1]
```diff
@@ -73,7 +73,7 @@ optlist,args = getopt.getopt(sys.argv[1:],"v")
 
 verbose = (assoc('-v',optlist)=='')
 truth_lines = open(args[0]).read().split('\n')
-actual_doc = html.fromstring(open(args[1]).read())
+actual_doc = html.parse(args[1])
```
Comment (stweil) [inline hocr-eval-lines]: @kba, do we need the same kind of change for the remaining seven code locations with `html.fromstring`, too?

Comment (kba) [inline hocr-eval-lines]: Yes, it makes the code more robust if lxml handles the string decoding.

**Pull request merged**